perl/highmon.pl

```diff
@@ -317,6 +319,15 @@ sub highmon_buffer_open
 		{
 			weechat::buffer_set($highmon_buffer, "localvar_set_no_log", "1");
 		}
+		# send "logger_backlog" signal if logging is enabled to display backlog
+		if (weechat::config_get_plugin("logging") eq "on")
```

Comment:
Hi,
You don't have to check if the log is enabled, the logger_backlog callback function already does that.
You can just send the signal in all cases, the logger plugin is OK with that.